diff --git a/qmake/CMakeLists.txt b/qmake/CMakeLists.txt
index 688314b736..8042dceee3 100644
--- a/qmake/CMakeLists.txt
+++ b/qmake/CMakeLists.txt
@@ -18,6 +18,7 @@ set_target_properties(QtLibraryInfo PROPERTIES
     INCLUDE_DIRECTORIES $<TARGET_PROPERTY:Qt::Core,INTERFACE_INCLUDE_DIRECTORIES>
     INCLUDE_DIRECTORIES $<TARGET_PROPERTY:Qt::CorePrivate,INTERFACE_INCLUDE_DIRECTORIES>
 )
+qt_set_common_target_properties(QtLibraryInfo)
 target_include_directories(QtLibraryInfo PUBLIC
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/library"
